lib/screens/profile/contact_support_public.dart

```diff
@@ -0,0 +1,292 @@
+import 'package:brunstadtv_app/router/router.gr.dart';
+import 'package:email_validator/email_validator.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:graphql/client.dart';
+import 'package:universal_io/io.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+import '../../components/contact_support/contact_support_failure.dart';
+import '../../components/contact_support/contact_support_sending_indicator.dart';
+import '../../components/contact_support/contact_support_success.dart';
+import '../../components/web/dialog_on_web.dart';
+import '../../components/general_app_bar.dart';
+import '../../graphql/client.dart';
+import '../../graphql/queries/send_support_email.graphql.dart';
+import '../../graphql/schema/mutations.graphql.dart';
+import '../../helpers/ui/btv_buttons.dart';
+import '../../helpers/ui/svg_icons.dart';
+import '../../providers/device_info.dart';
+import '../../theme/design_system/design_system.dart';
+
+import '../../l10n/app_localizations.dart';
+
+class ContactSupportPublicScreen extends HookConsumerWidget {
+  const ContactSupportPublicScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final nameController = useTextEditingController();
+    final emailController = useTextEditingController();
+    final messageController = useTextEditingController();
+    final includeDeviceInfo = useRef(true);
+    final isOnInputPage = useState(true);
+    final formKey = useMemoized(() => GlobalKey<FormState>());
+    final sendSupportEmailFuture = useState<Future<QueryResult>?>(null);
+    final sendSupportEmailSnapshot = useFuture(sendSupportEmailFuture.value);
+    final deviceInfo = ref.watch(deviceInfoProvider).valueOrNull;
+    useListenableSelector(nameController, () => nameController.text.isEmpty);
+    useListenableSelector(emailController, () => emailController.text.isEmpty);
+    useListenableSelector(messageController, () => messageController.text.isEmpty);
+
+    void onIncludeDeviceInfoChange(bool state) => includeDeviceInfo.value = state;
+
+    void onTryAgain() => isOnInputPage.value = true;
```

Comment:
`onTryAgain` and `onIncludeDeviceInfoChange` could be inlined where they are used to keep things more readable and clean.